source/ranges.tex

```diff
@@ -656,6 +657,29 @@
 type is integer-like.
 \end{note}
 
+\rSec2[range.prim.ssize]{\tcode{ranges::ssize}}
+
+\pnum
+The name \tcode{ssize} denotes
```

Comment:
There should be an index entry for ranges::ssize (it looks like nothing in this subclause is indexed though, so maybe fix them all together).

Reply:
@jwakely I added one for ranges::ssize, please open an edit issue for the rest.